syne_tune/optimizer/schedulers/searchers/regularized_evolution.py

```diff
@@ -0,0 +1,138 @@
+import copy
+
+from collections import deque
+from typing import Optional, Dict, List
+from dataclasses import dataclass
+
+from syne_tune.optimizer.schedulers.searchers import SearcherWithRandomSeed
+from syne_tune.search_space import Domain, Categorical, Float, Integer
+
+
+@dataclass
+class PopulationElement:
+    score: int = 0
+    config: dict = None
+
+
+class RegularizedEvolution(SearcherWithRandomSeed):
+    def __init__(
+            self,
+            configspace,
+            metric: str,
+            mode: str = 'min',
+            population_size: int = 100,
+            sample_size: int = 10,
+            points_to_evaluate: Optional[List[Dict]] = None,
+            random_seed: int = None
```

Comment:
This is not fully correct. I'd put **kwargs here, and pass that to the superclass.

The full story is that there is a random_seed_generator which is used to generate seeds from a master seed.